packages/apps/tools/src/components/Common/Layout/partials/Header/Header.tsx

```diff
@@ -30,6 +32,8 @@ const Header: FC<IHeaderProps> = () => {
 
   const currentTheme = theme === 'system' ? systemTheme : theme;
 
+  const isMediumScreen = useIsMatchingMediaQuery(`${breakpoints.sm}`);
```

Comment:
This should be a media query syntax with the media par, so probably you meant something like this
```suggestion
  const isMediumScreen = useIsMatchingMediaQuery(`(width <= ${breakpoints.sm})`);
```
or ```suggestion
  const isMediumScreen = useIsMatchingMediaQuery(`(min-width: ${breakpoints.sm})`);
```